packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -172,14 +190,38 @@ export class EditManager<TChangeset, TChangeFamily extends ChangeFamily<any, TCh
         );
     }
 
-    private getCommitsBetween(first: SeqNumber, last: SeqNumber | undefined): Commit<TChangeset>[] {
-        const firstIndex = this.getCommitIndex(first);
-        const lastIndex = last === undefined ? undefined : this.getCommitIndex(last);
-        return this.trunk.slice(firstIndex, lastIndex);
+    /**
+     * @param pred - The sequence number of the commit immediately before the commits of interest.
+     * @param last - The sequence number of the last commit of interest.
+     * @returns The trunk commits with sequence numbers greater than `pred` and smaller or equal to `last`,
+     * ordered in sequencing order.
+     */
+    private getCommitsAfterAndUpToInclusive(pred: SeqNumber, last: SeqNumber): Commit<TChangeset>[] {
+        // This check is not just a fast-path for the common case where no concurrent edits occurred,
+        // it also serves to handle the case where the `last` commit is the non-existent commit that the
+        // very first change on the document (and possibly others) will refer to.
```

Comment:
Arguably more precise wording

```suggestion
        // it also serves to handle the case where `last` represents the initial state before any commits.
```